### ISSUE TITLE
Enterprise Github failure

### ISSUE THREAD

**Issue by finchy** [dennisreimann/ioctocat#473]
Attempting to connect to a Enterprise Github account, with a Installed Certificate profile:

I get the error: 

> Authentication Failed
> The operation couldn't be completed.  (NSURLErrorDomain error -1012).

**Comment by dennisreimann** [comment]
Can you log in to the Enterprise instance in Safari? If so, it should work in iOctocat too.

**Comment by finchy** [comment]
Yes.  We have a custom certificate, but I imported it as per your instructions.

**Comment by vrtadmin** [comment]
Issue still exists.  What is the issue?  I've tried typing in the Domain in lots of different ways.
